evalml/model_understanding/graphs.py

```diff
@@ -454,26 +454,27 @@ def partial_dependence(pipeline, X, feature, grid_resolution=100):
             "positive" class.
 
     """
+    from evalml.pipelines.components.utils import scikit_learn_wrapped_estimator
```

Comment:
What's going on here: previously, we could pass our pipelines directly to scikit-learn's partial dependence method after a bit of finagling (adding attributes expected). This is no longer true because we're returning Woodwork data structures from predict and scikit-learn doesn't know how to handle that.

Thus, updating this code to wrap our pipeline in `scikit_learn_wrapped_estimator` (implemented for ensemble), updating the fields now on this wrapped object. 

IMO kinda cleaner than previous, where we had to delete these ugly attributes we added just for scikit-learn to operate since we don't care for the wrapped obj afterwards :d